flocx_market/.env

```diff
@@ -0,0 +1,2 @@
+DATABASE_URI=mysql+pymysql://flocx_market:qwerty123@127.0.0.1:3307/flocx_market
```

Comment:
It'd be much much better to use oslo config to get this; @ljmcgann and @DanNiESh should be able to give you pointers on how!

Reply:
Sure. We can use oslo config to replace the environment variables and hardcode lines @fvukelic

Reply:
This is not directly related to the use of oslo_config (or not); it's a docker-compose artifact. It should not be in this pull request.